src/services/itemMembership/plugins/MembershipRequest/entities/MembershipRequest.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  BaseEntity,
+  CreateDateColumn,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  Unique,
+  UpdateDateColumn,
+} from 'typeorm';
+
+import { Item } from '../../../../item/entities/Item';
+import { Member } from '../../../../member/entities/member';
+
+@Entity()
+@Unique('UQ_membership_request_item-member', ['item', 'member'])
```

Comment:
I really like the fact that we are naming the constraints and foreign key ! Do you think it is possible to also name PK and Idx ?

Reply:
Yes we can, I'll name the PK!